notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-1593_force_actuator_error_elevation.ipynb

```diff
@@ -0,0 +1,514 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    primary_FA_error = [
</pre><p>If might be more compact to write something like: </p><p><code>primary_FA_error = [f"primaryCylinderFollowingError{i}" for i in range(n_actuators)"]</code> </p><p>You can even consider adding the <code>[...]</code> directly to the query to get data from the Efd.   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/148/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Used your proposed syntax.